File: packages/nodejs/android/src/main/java/io/capawesome/capacitorjs/plugins/nodejs/classes/results/IsReadyResult.java
package io.capawesome.capacitorjs.plugins.nodejs.classes.results;

import androidx.annotation.NonNull;
import com.getcapacitor.JSObject;
import io.capawesome.capacitorjs.plugins.nodejs.interfaces.Result;

public class IsReadyResult implements Result {

    private final boolean ready;

    public IsReadyResult(boolean ready) {
        this.ready = ready;
    }

    @Override
    @NonNull
    public JSObject toJSObject() {
        JSObject result = new JSObject();
        result.put("ready", ready);
        return result;
    }
}

File: packages/nodejs/ios/Plugin/Classes/Results/IsReadyResult.swift
import Foundation
import Capacitor

@objc public class IsReadyResult: NSObject, Result {
    let ready: Bool

    init(ready: Bool) {
        self.ready = ready
    }

    @objc public func toJSObject() -> AnyObject {
        var result = JSObject()
        result["ready"] = ready
        return result as AnyObject
    }
}
